### ISSUE TITLE
Postgres error using Topic.visible.by_most_recent_post

### ISSUE THREAD

**Issue by jamescook** [rubysherpas/forem#20]
```
PGError: ERROR:  column "forem_topics.id" must appear in the GROUP BY clause or be used in an aggregate function
LINE 1: SELECT  "forem_topics".* FROM "forem_topics" INNER JOIN "for...
            ^
: SELECT  "forem_topics".* FROM "forem_topics" INNER JOIN "forem_posts" ON "forem_posts"."topic_id" = "forem_topics"."id"     WHERE "forem_topics"."hidden" = 'f' GROUP BY topic_id ORDER BY forem_posts.created_at DESC, forem_topics.id LIMIT 5
```

::Forem::Topic.visible.select("DISTINCT topic_id, forem_topics.*").limit(5).includes(:posts) works for me in postgres.

**Comment by radar** [comment]
Would you be able to submit a patch that fixes this issue please?

**Comment by jamescook** [comment]
Yes, I will

**Comment by parndt** [comment]
Solved by pull request #21.
